st2common/st2common/cmd/validate_api_spec.py

```diff
@@ -22,8 +22,8 @@
 from __future__ import absolute_import
 import os
 
+import prance
```

Comment:
I understand the help `prance` offers with openapi, but I'm a little concerned with the activity of the project.  It claims to be tested against py36-py38 and hasn't received an update since 26 Nov 2021.  Perhaps this is a sign of its code maturity, but it would be unfortunate to fall into the same trap we did with `udatetime` where it became more of a hindrance than a help.  If you're of the opinion that the project is alive and active, I'm OK with adding it as a dependency.

Reply:
OK, ignore me, it's not an addition, it's already a dependency, sorry for the noise.  But my concern still stands, we may need to look for an active project if `prance` has been abandoned.

Reply:
Agreed. There are a bunch of things I'd like to improve around the openapi spec. Getting rid of `prance` sounds great to me. We're also using openapi/swagger 2, which means we can't use a lot of newer tooling that only supports openapi 3+. So much to do...